build.ps1

```diff
@@ -122,13 +122,14 @@ Get-ChildItem $ReferenceDocset -Directory -Exclude 'docs-conceptual', 'mapping',
         }
         catch {
             $allErrors += $_
+            $_
         }
     }
 
 }
 
 # If the above block, produced any errors, throw and fail the job
 if ($allErrors) {
-    $allErrors
+    # $allErrors
```

Comment:
I would still keep this to see all the errors together or at least a count so that you can do a `$errors[-4]` for example.